db.go

```diff
@@ -528,6 +530,47 @@ func (db *DB) Close() error {
 	return merr.Err()
 }
 
+// DisableCompactions disables compactions.
+func (db *DB) DisableCompactions() error {
```

Comment:
As long as there's no possible error and it's not required by an interface, no need to return an error.

That said, we'll need to have a way for these operations to time out while trying to acquiring a lock (via passing a context) or have them enforce quick acquisition of locks, which is probably not possible due to long running queries.
Again somewhat circles back to #84 – we can look at this down the road.